Pipfile

```diff
@@ -0,0 +1,51 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+aiohttp = "==3.7.4.post0"
+async-timeout = "==3.0.1"
+attrs = "==21.2.0"
+beautifulsoup4 = "==4.9.3"
+certifi = "==2021.5.30"
+chardet = "==4.0.0"
+dateparser = "==1.0.0"
+et-xmlfile = "==1.1.0"
+idna = "==2.10"
+multidict = "==5.1.0"
+numpy = "==1.20.3"
+openpyxl = "==3.0.7"
+pandas = "==1.2.4"
+pyqtgraph = "==0.12.1"
+python-binance = "==1.0.12"
+python-dateutil = "==2.8.2"
+python-telegram-bot = "==13.7"
+pytz = "==2021.1"
+regex = "==2021.7.6"
+requests = "==2.26.0"
+six = "==1.16.0"
+soupsieve = "==2.2.1"
+tornado = "==6.1"
+typing-extensions = "==3.10.0.0"
+tzlocal = "==2.1"
+ujson = "==4.0.2"
+urllib3 = "==1.26.6"
+websockets = "==9.1"
+yarl = "==1.6.3"
+APScheduler = "==3.6.3"
+PyQt5 = "==5.15.4"
+PyQt5-Qt5 = "==5.15.2"
+PyQt5-sip = "==12.9.0"
+
+[dev-packages]
+flake8 = "*"
+isort = "*"
+pytest = "*"
+pylint = "*"
+pytest-cov = "*"
+pytest-socket = "*"
+pre-commit = "*"
+
+[requires]
+python_version = "3.8"
```

Comment:
Let's drop this - this limits the Python version that can be used. E.g. I am on Python 3.9 meaning people will have to have that resulting in an extra step.